<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfAboutDialog</class>
 <widget class="QDialog" name="WkfAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="additionalLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Additional Info Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pluginVersionLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Warlock Plugin Version Number :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="buildLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Built on :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="artworkPushButton">
       <property name="text">
        <string>Artwork Attribution...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="licensePushButton">
       <property name="text">
        <string>3rd Party Licenses...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="systemInfoPushButton">
       <property name="text">
        <string>System Info...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
